aapl_IEX_model.ipynb

```diff
@@ -0,0 +1,1659 @@
+{
```

Comment:
<p>use <code>get_stock_data</code>  instead of <code>get_raw_data</code>  (they return the same thing , <code>get_stock_data</code>   gets  ticker eg <code>"AAPL"</code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/alongouldman/stock_market_neural_nets/pull/1/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Reply:
<p>i delete the function - there is no real use for it.</p>